### ISSUE TITLE
Wrong titel for number (sungrow)

### ISSUE THREAD

**Issue by smurfb** [biemond/solaredge.modbus#67]
With sungrow sth.8 the field "Grid Export Day Yield" number is in fact total grid import (called Total Purchased Energy in isolar). 

Thanks for adding sungrow!

**Comment by biemond** [comment]
hey, 

thanks will check it.

**Comment by biemond** [comment]
yeah saw I fixed it already with  52f9e2b40595ff8a0e27a9d51c9f0619ff014a92

a recent one should have it
